network_path/manifest.json

```
@@ -0,0 +1,46 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "3913cf32-d869-4852-beda-9e3e9d9e2feb",
+  "app_id": "network-path",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Network Path integration collects traceroute-like data.",
```

```suggestion
    "description": "Network Path integration collects traceroute data.",
```

Just suggesting to fix the description here too as it removes confusion